src/components/modal/Modal.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import "./Modal.scss"
+import { useState } from 'react'
+
+const Modal = () => {
+
+    const [modal, setModal] = useState(false)
+
+    const toggleModal = () => {
+        setModal(!modal)
+    }
+
+    return (
+        <>
```

Comment:
Add name to top level container to make it semantic

Reply:
Got it, will do